lib/companion/application.ex

```diff
@@ -12,9 +15,23 @@ defmodule Companion.Application do
       # Start the Ecto repository
       supervisor(Companion.Repo, []),
       # Start the endpoint when the application starts
-      supervisor(CompanionWeb.Endpoint, [])
-      # Start your own worker by calling: Companion.Worker.start_link(arg1, arg2, arg3)
-      # worker(Companion.Worker, [arg1, arg2, arg3]),
+      supervisor(CompanionWeb.Endpoint, []),
+      # Jobs
+      {Honeydew.EctoPollQueue,
+       [
+         :process_opt_out,
+         schema: OptOut,
```

Comment:
Maybe pull this `Honeydew.EctoPollQueue` config out into a separate function (or module attribute or something) and just reference it here? Having a big structure in the middle of a list of one-line child process definitions is a bit disorienting and the `Honeydew.Workers` at the bottom gets a bit lost.